qemu/tests/block_hotplug.py

```diff
@@ -121,8 +142,13 @@ def unplug_device(vm, get_disk_cmd, device):
                         test.fail(err)
                     else:
                         controller_list.append(controller)
+                
+                if drive_cmd_type == "drive_add":
+                    drive = qdevices.QHPDrive("block%d" % num)
+                    drive.set_param("if","none")
```

Comment:
and here you miss a white space after `,`, therefor it should be `drive.set_param("if", "none")`.